project/pyproject.toml.jinja

```diff
@@ -15,12 +15,12 @@ keywords = []
 packages = [ { include = "[[ python_package_import_name ]]", from = "src" } ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2,<4.0"
```

Comment:
Are you sure this is required? Based on this article https://iscinumpy.dev/post/bound-version-constraints/, I'd really like to avoid that. Current Black setup.py does not seem to cap it, so we should not need to cap either: https://github.com/psf/black/blame/main/setup.py#L97

Reply:
i agree with that but requirement to not have python 4 have to remain due to black package

Reply:
git-changelog has a more recent version that removed the upper bound. We should simply remove all upper bounds on the dependencies. Let me do that :slightly_smiling_face:

Reply:
Oh well, dependencies that are not under my control also set this cap. So indeed we'll have to leave it for now :slightly_smiling_face: I'll still remove upper bounds on the deps.